.github/workflows/build-cleaner.yaml

```diff
@@ -0,0 +1,23 @@
+---
+on:
+  pull_request:
```

Comment:
On push or pull_request? I would assume push to the main

Reply:
Yeah, good point!